manifest.toml

```diff
@@ -0,0 +1,100 @@
+#:schema https://raw.githubusercontent.com/YunoHost/apps/master/schemas/manifest.v2.schema.json
+
+packaging_format = 2
+
+id = "weblate"
+name = "Weblate"
+description.en = "Translation platform using Git and Python"
+description.fr = "Plateforme de traduction utilisant Git et Python"
+
+version = "5.3.1~ynh1"
+
+maintainers = ["YunoHost Contributors"]
+
+[upstream]
+license = "AGPL-3.0"
+website = "https://weblate.org"
+demo = "https://hosted.weblate.org/"
+admindoc = "https://docs.weblate.org/"
+userdoc = "https://docs.weblate.org/"
+code = "https://github.com/WeblateOrg/weblate"
+cpe = "cpe:2.3:a:weblate:weblate"
+
+[integration]
+yunohost = ">= 11.2.0"
+architectures = "all"
+multi_instance = true
+ldap = false
+sso = false
+
+disk = "1G"
+ram.build = "3G"
+ram.runtime = "3G"
+
+[install]
+    [install.domain]
+    type = "domain"
+
+    [install.path]
+    type = "path"
+    default = "/weblate"
+
+    [install.init_main_permission]
+    type = "group"
+    default = false
+
+    [install.admin]
+    type = "user"
+
+    [install.password]
+    type = "password"
+
+    [install.used_forge]
+    ask.en = "Choose the forge you use"
+    ask.fr = "Choisissez la forge que vous utilisez"
+    type = "select"
+    choices = ["GitHub", "GitLab"]
```

Comment:
Not sure you know, but this can be a dict. For example:

```
choices.github = "Github"
choices.gitlab = "GitLab"
choices.another_forge = "Some string that will be translatable in the future"
```

That way, the setting / bash variable is the key (so without caps / spaces), I find it easier / less error prone.
That’s not mandatory though, it’s OK as is :)

Reply:
Oooh nice!